cves/kernel/CVE-2020-14386.yml

```diff
@@ -343,9 +380,11 @@ stacktrace:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  any_stacktraces:
-  stacktrace_with_fix:
-  note:
+  any_stacktraces: false
+  stacktrace_with_fix: false
+  note: |
+    Looking at the bug reports, github, MITRE, NVD, and blogs, I found no
```

Comment:
Looking through the NVD listing, I cam across a stack trace in the announcement from Or Cohen explaining the vulnerability.

Reply:
somehow missed that, thanks